compiler/test/formatter_inputs/function_params.gr

```diff
@@ -4,10 +4,13 @@ let unit_arg = () => 3
 
 let two_args = (x,y) => 4
 
+@disableGC
```

Comment:
This is maybe somewhat meta, but could we instead swap the signature of this function to use non-unsafe values? People often look at tests for examples, and I don't want them getting ideas 😛

Reply:
Good point !

Reply:
Fixed.